src/hotspot/cpu/riscv/riscv.ad

```diff
@@ -7506,6 +7506,56 @@ instruct roundD_reg(fRegD dst, fRegD src, immI rmode, iRegLNoSp tmp1, iRegLNoSp
   ins_pipe(pipe_class_default);
 %}
 
+// Copysign and signum intrinsics
+
+instruct copySignD_reg(fRegD dst, fRegD src1, fRegD src2, immD0 zero) %{
```

Comment:
A more simpler `immD zero` will do here which is similar with the x86 counterpart.